### PR TITLE
Fixed yaml to be accurate (hopefully)

### PR DESCRIPTION
**Pull request by SushrutHarness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Purpose

What is the motivation for this PR? What changes does it include? Include as much or as little detail as needed.

Correct incorrect yaml in google functions file.

If this submission relates to a Jira or GitHub issue, include the issue ID in the title.

Internal Harness contributors - Please edit this summary to include preview links after the preview build finishes.

## Merge requirements

- [X] Successful preview build.
- [ ] Code owner review.
- [X] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md [hunk 1]
```diff
@@ -227,10 +227,10 @@ function:
   buildConfig:
     runtime: python38
     entryPoint: my_function
+    environmentVariables:
```
Comment (guilhermezanini-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md:230]: According to [Google Docs](https://cloud.google.com/functions/docs/reference/rpc/google.cloud.functions.v2#buildconfig), this one should be in snake case: `environment_variables`

**Review by guilhermezanini-harness — CHANGES_REQUESTED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md (hunk 1 above)
Comment (guilhermezanini-harness) [reply docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md:230]: Also in snake case the `buildConfig` attribute.

**Review by guilhermezanini-harness — CHANGES_REQUESTED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md (hunk 1 above)
Comment (guilhermezanini-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md:230]: According to [Google Docs](https://cloud.google.com/functions/docs/reference/rpc/google.cloud.functions.v2#buildconfig), this one should be in snake case: environment_variables

The same applies to `buildConfig` and `entryPoint`.

**Review by guilhermezanini-harness — CHANGES_REQUESTED** [review]

File: docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md [hunk 2]
```diff
@@ -290,29 +291,30 @@ function:
   description: "Using Secret Environment Variables"
   region: "us-east1"
   runtime: "nodejs16"
-  entryPoint: myFunction
+  entry_point: myFunction
```
Comment (anshul-harness) [inline docs/continuous-delivery/deploy-srv-diff-platforms/google-functions.md:294]: This looks incorrect to me. Please verify. This does not match the schema https://cloud.google.com/functions/docs/reference/rpc/google.cloud.functions.v2#function

**Review by guilhermezanini-harness — APPROVED** [review]
Everything looks good.

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request merged**